### PR TITLE
Change license to AGPLv3

### PR DESCRIPTION
**Pull request by alextran1502**
REF: https://github.com/immich-app/immich/discussions/7023

This PR changes the license of the Immich repo from `MIT` to `GNU AFFERO GENERAL PUBLIC LICENSE Version 3` or `AGPLv3`.

image

> [!IMPORTANT]
> ### Version 1.94.1 (https://github.com/immich-app/immich/commit/d7437d31d1796f1bff906fe0f3e0bb336f7b344e) and prior are still under MIT.
> ### Version 1.95.0 (https://github.com/immich-app/immich/commit/dca1bd22df067104129ca4f30274e4f8c9f3a349) and onward will be under AGPLv3.
> ### There will be no CLA enforced

### CODE REVIEW

**Review by jrasm91 — APPROVED** [review]

**Review by adamantike — APPROVED** [review]

**Review by shenlong-tanwen — APPROVED** [review]

**Comment by adamantike** [comment]
There are more mentions to the MIT license, in case they want to be replaced in this PR:

```
$ rg --glob '!**/LICENSE' -w MIT
README_tr_TR.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README_es_ES.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="Licencia: MIT"></a>

README_ru_RU.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README_ca_ES.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=Llicència&logoColor=000000&labelColor=ececec" alt="Llicència: MIT"></a>

open-api/typescript-sdk/package-lock.json
10:      "license": "MIT",

open-api/typescript-sdk/package.json
21:  "license": "MIT",

web/package-lock.json
67:      "license": "MIT",

README_nl_NL.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README_ja_JP.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README_de_DE.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="Lizenz: MIT"></a>

README_it_IT.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README_fr_FR.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

cli/package-lock.json
10:      "license": "MIT",
47:      "license": "MIT",

docs/src/pages/index.tsx
74:        <p>This project is available under MIT license.</p>

README_ko_KR.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

README_zh_CN.md
3:  <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/license-MIT-green.svg?color=3F51B5&style=for-the-badge&label=License&logoColor=000000&labelColor=ececec" alt="License: MIT"></a>

cli/package.json
10:  "license": "MIT",

docs/docusaurus.config.js
164:        copyright: `Immich is available as open source under the terms of the MIT License.`,
```

**Review by martyfuhry — APPROVED** [review]

**Comment by alextran1502** [comment]
@adamantike Good catch, thank you!

**Review by mertalev — APPROVED** [review]

**Review by brighteyed — APPROVED** [review]

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (mertalev) [inline LICENSE-MIT:21]: Nit: add a new line

**Review by martabal — APPROVED** [review]

**Review by etnoy — APPROVED** [review]
Lgtm

**Review by danieldietzler — APPROVED** [review]

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,661 @@
...
+get its source.  For example, if your program is a web application, its
+interface could display a "Source" link that leads users to an archive
+of the code.  There are many ways you could offer source, and different
+solutions will be better for different programs; see section 13 for the
+specific requirements.
+
+  You should also get your employer (if you work as a programmer) or school,
+if any, to sign a "copyright disclaimer" for the program, if necessary.
+For more information on this, and how to apply and follow the GNU AGPL, see
+<https://www.gnu.org/licenses/>.
```
Comment (bo0tzz) [inline LICENSE:661]:
```suggestion

```

This section is not part of the license

File: cli/LICENSE [hunk 3]
```diff
@@ -1,21 +1,661 @@
...
+get its source.  For example, if your program is a web application, its
+interface could display a "Source" link that leads users to an archive
+of the code.  There are many ways you could offer source, and different
+solutions will be better for different programs; see section 13 for the
+specific requirements.
+
+  You should also get your employer (if you work as a programmer) or school,
+if any, to sign a "copyright disclaimer" for the program, if necessary.
+For more information on this, and how to apply and follow the GNU AGPL, see
+<https://www.gnu.org/licenses/>.
```
Comment (bo0tzz) [inline cli/LICENSE:661]:
```suggestion

```

**Review by bo0tzz — APPROVED** [review]

**Review by zackpollard — APPROVED** [review]

File: LICENSE-MIT
Comment (fyfrey) [inline LICENSE-MIT:1]: following the discussion on discord, we should remove this file to not cause confusion (e.g. being dual-licensed as MIT)

**Review by fyfrey — APPROVED** [review]

**Pull request merged**